README.md

```diff
@@ -216,6 +216,8 @@ If any value is set for `NHAIS_MONGO_HOST` then the following properties will be
 | NHAIS_MONGO_USERNAME             |         | (Optional) Mongodb username. If set then password must also be set.
 | NHAIS_MONGO_PASSWORD             |         | (Optional) Mongodb password
 | NHAIS_MONGO_OPTIONS              |         | (Optional) Mongodb URL encoded parameters for the connection string without a leading ?
+| NHAIS_MONGO_TTL                  | P30D    | (Optional) Time-to-live value for inbound and outbound state collection documents in ISO-8601 format
+| NHAIS_COSMOS_DB_ENABLED          | false   | (Optional) If true NHAIS will use Cosmos DB implementation instead of Mongo DB
 
 If no value is set for `NHAIS_MONGO_HOST` then the following properties are used:
 
```

Comment:
Could you include the following in a new section after the environment variables?

```
#### Time-to-live Indexes

The adaptor creates TTL (time to live) indexes on the `outboundState` and `inboundState` collections to automatically 
remove old documents. The variable `NHAIS_MONGO_TTL` describe above controls the duration. There are differences between
how TTL indexes work between mongodb and Azure Cosmos. If using Cosmos the `NHAIS_COSMOS_DB_ENABLED` flag must be true.

**TTL Indexes in Mongo**

The property `translationTimestamp` is indexed. For outbound, this is the timestamp when the adaptor translates FHIR 
into EDIFACT and is the timestamp enclosed in the EDIFACT interchange sent to NHAIS. For inbound, this is the 
timestamp enclosed within the EDIFACT interchange received from NHAIS.

**TTL Indexes in Cosmos**

[TTL Index in Cosmos](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-time-to-live) are limited in that only 
a specific `_ts` property may have this index. The _ts property "is a system generated property (that) specifies the 
last updated timestamp of the resource". ([Reference](https://docs.microsoft.com/en-us/rest/api/cosmos-db/databases))

For outbound, each document is "last updated" when the adaptor processes the inbound RECEP for that transaction. For
inbound each document is "last updated" after publishing the FHIR message to the inbound GP System message queue.

The impact is that documents in Cosmos may live slightly longer than those stored in Mongo. 
```